pydm/application.py

```diff
@@ -132,13 +132,27 @@ def load_ui_file(self, uifile, macros=None):
       f = uifile
     return uic.loadUi(f)
     
+  def __sanity_check_pyqt(self, cls):
+    def check(attr):
```

Comment:
When I do `from PyQt4.QtCore import QObject, pyqtSlot, pyqtSignal`, I don't get a warning.  We could maybe do something like:
`````````````````````
for module in dir(cls):
    check(module)
`````````````````````
And then change check() to just look for 'PyQt4' or 'PyQt5' in `__file__` for the module.  This might be heavier than we want, should we really be investigating every imported module?